.github/workflows/cd-dgraph.yml

```diff
@@ -39,9 +39,9 @@ jobs:
       - name: Make Linux Build
         run: make dgraph DGRAPH_VERSION=${{ env.DGRAPH_RELEASE_VERSION }}
       - name: Generate SHA for Linux Build
-        run: sha256sum dgraph/dgraph | cut -c-64 > dgraph/dgraph-linux-amd64.sha256
+        run: cd dgraph && sha256sum dgraph | cut -c-64 > dgraph-linux-amd64.sha256
```

Comment:
@dshekhar95 small request, I should have made this clear on the issue. I just realized this, so I am raising it on this PR. 

could you rename `dgraph-linux-amd64.sha256` to `dgraph-checksum-linux-amd64.sha256` please? This was a typo that had gotten in, from our release work. Apparently the old team was using `dgraph-checksum-*` and those references are made in other places which would break, if we don't make this fix. 

Could you squeeze that change in this PR itself? (again apologies for late feedback)

Also, you don't have to test that - I did look at the artifacts from your branch. It looks right. If you make a change on line#42, you will also have to change that in `upload artifact step`